skopt/acquisition.py

```diff
@@ -34,23 +34,40 @@ def _gaussian_acquisition(X, model, y_opt=None, acq_func="LCB",
     kappa = acq_func_kwargs.get("kappa", 1.96)
 
     # Evaluate acquisition function
+    per_second = acq_func.endswith("ps")
+    if per_second:
+        model, time_model = model.estimators_
+
     if acq_func == "LCB":
-        return gaussian_lcb(X, model, kappa, return_grad)
+        func_and_grad = gaussian_lcb(X, model, kappa, return_grad)
+        if return_grad:
+            acq_vals, acq_grad = func_and_grad
+        else:
+            acq_vals = func_and_grad
 
-    elif acq_func in ["EI", "PI"]:
-        if acq_func == "EI":
+    elif acq_func in ["EI", "PI", "EIps", "PIps"]:
+        if acq_func in ["EI", "EIps"]:
             func_and_grad = gaussian_ei(X, model, y_opt, xi, return_grad)
         else:
             func_and_grad = gaussian_pi(X, model, y_opt, xi, return_grad)
 
         if return_grad:
-            return -func_and_grad[0], -func_and_grad[1]
+            acq_vals = -func_and_grad[0]
+            acq_grad = -func_and_grad[1]
         else:
-            return -func_and_grad
+            acq_vals = -func_and_grad
 
     else:
         raise ValueError("Acquisition function not implemented.")
```

Comment:
I have added detailed error messages in the Optimizer instance. It's unlikely that anyone will use this private function directly.